### DISCUSSION TITLE
multiple login bug

### DISCUSSION THREAD

**Discussion by Fillox112** [betfinio/public#10] [Bugs and Reports]
When a user logs in on a PC via QR code, has MetaMask on their phone, and connects multiple wallets to MetaMask, every time the user clicks anywhere, it automatically switches back to the first logged-in account.

When I use the first logged-in account, everything works fine. However, if I switch to account 2 on my phone and click anywhere, account 2 switches back to account 1, which was logged in first.

Is this a Betfin or MetaMask bug?

**Comment by bf-monster** [comment]
Hey. We will investigate this issue. Thank you for report

**Comment by bf-gabagool** [comment]
The issue has been fixed. Thank you for letting us know!
